ors-engine/src/main/java/org/heigit/ors/routing/APIEnums.java

```diff
@@ -420,6 +420,8 @@ public enum Languages {
         NE_NP("ne-np"),
         NL("nl"),
         NL_NL("nl-nl"),
+        NO("no"),
+        NO_NB("no-nb"),
```

Comment:
Hey, I am not sure what is correct, but according to https://www.localeplanet.com/icu/nb-NO/index.html, `nb-no` would be used here?

Reply:
Looks like that is the most common. I have updated it in the file and renamed the file with the translation to match this.